doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -196,12 +187,11 @@ of types inside the syntactic class :token:`term`.
 Qualified identifiers and simple identifiers
 --------------------------------------------
 
-.. insertgram qualid field
+.. insertprodn qualid field_ident
 
-.. productionlist:: coq
-   qualid : `qualid` `field`
-          : `ident`
-   field : .`ident`
+.. prodn::
+   qualid ::= @ident {* @field_ident }
+   field_ident ::= .@ident
 
 *Qualified identifiers* (:token:`qualid`) denote *global constants*
 (definitions, lemmas, theorems, remarks or facts), *global variables*
```

Comment:
This paragraph contains a reference to `field`, which was renamed `field_ident`.